versioned_docs/version-v2.18.x/user-guide/configure-apiml-zosmf-workflow-2-18.md

```diff
@@ -0,0 +1,122 @@
+post-deployment workflow for mounting, which is required.
+
+After you successfully performed the **Zowe Mount** workflow, you can start the **Stand-alone Zowe API ML Configuration** workflow.
+When you select this workflow from the list of Workflow Definition Files, the following screen displays: 
+
+![Stand-alone Zowe API ML Configuration workflow](../images/zosmf/workflow-APIMLConfiguration.png)
+
+You can see the workflow details by expanding the **Workflow Details** panel.
+This workflow has three main steps:
+
+[1. Define variables](#1-define-variables)  
+[2. Create configuration](#2-create-configuration)  
+[3. Perform Zowe installation](#3-perform-zowe-installation)
+
+### 1. **Define variables**
+
+This workflow step includes the list of Zowe variables.
+When you expand **Define variables**, the following screen displays: 
+
+![Step1.1: Define the main variables](../images/zosmf/workflow-APIMLdefineMainVariablesV2.png)
+
+First, define the main variables for your configuration.
+
+After you complete defining the main variables for your configuration, define all ports for automatically enabled API ML services.  
+
+![Step1.2: Ports](../images/zosmf/workflow-APIMLdefinePortsV2.png)
+
+Perform the following steps to execute each sub-step individually:
+
+1. Click the title of the sub-step.
+2. Select the **Perform** tab.
+3. Review the step contents and update the variables that are marked by a red asterisk based on your mainframe environment.
+4. Select **Next**. 
+5. Repeat the previous two steps to complete all items until the **Finish** option is available.
+
+:::note
+Basic validation is supported in many fields. Validation includes proper path structure, dataset name conventions, or numeric size.
+The workflow, however, does not check, for example, whether a target dataset exists, or a directory has sufficient space.
+:::
+
+After both sub-steps, **Define the main variables** and **Ports** are completed, the step **Define variables** is marked as Complete.
+
+### 2. **Create configuration**
+
+This step creates a configuration zowe.yaml file with the variable setup that was defined in the previous step. 
+In this step you can review your configurations and, if necessary, make further changes directly in the JCL. 
+When you are done, click **Finish**. The zowe.yaml file is ready, and the step is marked as Complete.
+
+:::note
+The **Create configuration** step is mandatory. A valid zowe.yaml is required to execute the **Zowe installation** step.
+:::
+
+### 3. **Perform Zowe Installation**
+
+This step consumes the zowe.yaml configuration file you created in the previous step. It contains two sub-steps:
+
+1. **Run the Zowe install** to run the `zwe install` command.
+2. **Run the Zowe init** to run `zwe init mvs` and `zwe init stc` commands.
+
+:::note
+If you did not set up security prior this installation, you can alternatively submit the `ZWESECUR` JCL. For details, see [Configuring security](configuring-security.md).
```

Comment:
Can we remove the word "alternatively"? We are not describing any other way, so there it would seem that "alternatively" does not mean anything in this sentence.